### PR TITLE
refactor: extract framework aggregator from web codegen

### PR DESCRIPTION
**Pull request by Shikanime**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature (please, look at the "Scope of the project" section in the README.md file)
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #322

## Does this PR introduce a breaking change?

- [x] Yes
- [ ] No

### CODE REVIEW

File: projects/angular-codegen/package.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "@xlayers/angular-codegen",
+  "version": "0.0.1",
+  "peerDependencies": {
+    "@angular/common": "^7.2.0-rc.0",
+    "@angular/core": "^7.2.0-rc.0",
```
Comment (Jefiozie) [inline projects/angular-codegen/package.json]:
```suggestion
    "@angular/core": "^8.2.0",
```

File: projects/lit-element-codegen/package.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "@xlayers/lit-element-codegen",
+  "version": "0.0.1",
+  "peerDependencies": {
+    "@angular/common": "^7.2.0-rc.0",
+    "@angular/core": "^7.2.0-rc.0",
```
Comment (Jefiozie) [inline projects/lit-element-codegen/package.json]:
```suggestion
    "@angular/core": "^8.2.0",
```

File: projects/stencil-codegen/package.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "@xlayers/stencil-codegen",
+  "version": "0.0.1",
+  "peerDependencies": {
+    "@angular/common": "^7.2.0-rc.0",
+    "@angular/core": "^7.2.0-rc.0",
```
Comment (Jefiozie) [inline projects/stencil-codegen/package.json]:
```suggestion
    "@angular/core": "^8.2.0",
```

File: projects/web-component-codegen/package.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "@xlayers/web-component-codegen",
+  "version": "0.0.1",
+  "peerDependencies": {
+    "@angular/common": "^7.2.0-rc.0",
+    "@angular/core": "^7.2.0-rc.0",
```
Comment (Jefiozie) [inline projects/web-component-codegen/package.json]:
```suggestion
    "@angular/core": "^8.2.0",
```

**Comment by Jefiozie** [comment]
@manekinekko  @Shikanime , FYI in my tests I found out that the StackBlitz export is not correct anymore. I will file a separate issue for this.

**Comment by manekinekko** [comment]
Let's chat about this PR in our sync.

**Comment by Jefiozie** [comment]
hey, @Shikanime I made a fix for issue #333 could you update your branch with master?
Also, this PR is ready to be merged do you think we are good to go or do you want some more feedback?

**Comment by Shikanime** [comment]
@Jefiozie I will merge it by tomorrow at the latest 👍

**Pull request merged**